libs/service-portal/access-control/delegations/src/screens/AccessControl.tsx

```diff
@@ -0,0 +1,104 @@
+import { useHistory } from 'react-router-dom'
+import { useLocation } from 'react-use'
+import * as kennitala from 'kennitala'
+import { Box, Tabs } from '@island.is/island-ui/core'
+import {
+  IntroHeader,
+  m,
+  ServicePortalPath,
+} from '@island.is/service-portal/core'
+import { useLocale, useNamespaces } from '@island.is/localization'
+import { useAuth } from '@island.is/auth/react'
+import { isDefined } from '@island.is/shared/utils'
+import { DelegationsIncoming } from '../components/delegations/incoming/DelegationsIncoming'
+import { DelegationsOutgoing } from '../components/delegations/outgoing/DelegationsOutgoing'
+import { Features, useFeatureFlag } from '@island.is/react/feature-flags'
+
+const TAB_DELEGATION_OUTGOING_ID = '0'
+const TAB_DELEGATION_INCOMING_ID = '1'
+const DELEGATIONS_INCOMING_PATH = `${ServicePortalPath.MinarSidurPath}${ServicePortalPath.AccessControlDelegationsIncoming}`
+
+const AccessControl = () => {
+  useNamespaces(['sp.settings-access-control', 'sp.access-control-delegations'])
+  // TODO: Remove feature flag when incoming delegations are ready
+  const incomingFeatureFlag = useFeatureFlag(
+    Features.incomingDelegationsV2,
+    false,
+  )
+  const { formatMessage } = useLocale()
+  const { userInfo } = useAuth()
+  const history = useHistory()
+  const location = useLocation()
+  const isDelegationIncoming = location.pathname === DELEGATIONS_INCOMING_PATH
+
+  const isCompany = userInfo
+    ? kennitala.isCompany(userInfo.profile.nationalId)
+    : false
+
+  const tabChangeHandler = (id: string) => {
+    const url =
+      id === TAB_DELEGATION_INCOMING_ID
+        ? ServicePortalPath.AccessControlDelegationsIncoming
+        : ServicePortalPath.AccessControlDelegations
+
+    // Make sure not to add to history stack the same route twice in a row
+    if (url !== location.pathname) {
+      history.push(url)
+    }
+  }
+
+  // Only show outgoing delegation when user is logged in on behalf of someone else, i.e. some delegation.
+  const onlyOutgoingDelegations = isDefined(userInfo?.profile?.actor)
```

Comment:
~Maybe we should have used and checked a separate scope for the incoming delegations logic. Maybe even `@island.is/internal`? Instead of doing token sniffing like this, both in API and UI.

cc @saevarma